src/operators/repeat.ts

```diff
@@ -76,15 +72,15 @@ class MoreRepeatSubscriber<T> extends Subscriber<T> {
     super(null);
   }
 
-  _next(value: T) {
+  _next(value: T): void {
     this.parent.destination.next(value);
   }
 
-  _error(err: any) {
+  _error(err: any): void {
     this.parent.destination.error(err);
```

Comment:
Very correct. I thought in a way of deterministic using fixed scalar values all time. Then this line requires test coverage expanded, I'll update that.